llm-router/SPEC.md

```
@@ -0,0 +1,306 @@
+classifiers      — complexity classifier configs {id, model, threshold, categories}
+ab_tests         — A/B test definitions {id, name, variants: [{model, weight}], metric, status}
+ab_events        — recorded outcomes {test_id, variant, quality_score, latency_ms, cost_usd}
+routing_log      — decision audit trail {timestamp, request_id, policy_matched, model_selected, reason}
+model_health     — provider status {model → {available, latency_p99, error_rate, last_checked}}
+```
+
+## Functions (18)
+
+### Core Routing
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)